packages/ui-extensions-dev-console/src/sections/Extensions/components/ExtensionRow/ExtensionRow.tsx

```diff
@@ -54,7 +74,7 @@ export function ExtensionRow({uuid}: Props) {
             showModal
               ? {
                   url: extension.development.root.url,
-                  type: extension.surface,
+                  type: isUnifiedPOSExtension() ? 'point_of_sale' : extension.surface,
```

Comment:
Since extension.surface is on the root object, in extensions using the unified package, extension.surface will always be "all". This new function checks to see if the extension is a unified extension AND has POS targets. When we add support for Admin, we'll likely have to tweak that logic